packages/cli/src/index.ts

```diff
@@ -18,7 +18,7 @@ export async function main(medplumClient: MedplumClient, argv: string[]): Promis
   const clientId = process.env['MEDPLUM_CLIENT_ID'];
   const clientSecret = process.env['MEDPLUM_CLIENT_SECRET'];
   if (clientId && clientSecret) {
-    await medplum.startClientLogin(clientId, clientSecret);
+    await medplum.requestAccessToken(clientId, clientSecret);
```

Comment:
Should we just replace the implementation of `startClientLogin` with your new `requestAccessToken`?

Reply:
Or, in `fetchTokens`, do something like this:

```ts
if (clientId && clientSecret) {
      headers['Authorization'] = `Basic ${encodeBase64(clientId + ':' + clientSecret)}`;
}
```

I haven't fully thought that through, but it might be less surface area than new code path.

Reply:
I have something possibly similar to what you might be thinking
https://github.com/medplum/medplum/pull/2156 (accidentally created a pull request)